clearwater-diags-monitor/usr/share/clearwater/bin/gather_diags_and_report_location

```diff
@@ -32,21 +32,18 @@ fi
 
 DIAGS_END_LOCATION=${CLEARWATER_USER_HOME}/ftp/dumps/
 
-if [ $# -ne 0 ]
+/usr/share/clearwater/bin/gather_diags $@
```

Comment:
I'd change this to `. /usr/share/clearwater/bin/gather_diags $@` so that $0 doesn't change.